master/reference/calicoctl/setup/etcdv2.md

```diff
@@ -45,15 +45,15 @@ See the table below for details on the etcdv2 specific environment variables.
 
 ## Complete list of etcdv2 connection configuration
 
-| Spec field      | Environment       | Description                                | Examples
-|-----------------|----------------------------------------------------------------|----------
-| datastoreType   | DATASTORE_TYPE    | Indicates the datastore to use (optional, defaults to etcdv2) | etcdv2
-| etcdEndpoints   | ETCD_ENDPOINTS    | A comma separated list of etcd endpoints (optional, defaults to http://127.0.0.1:2379) | http://etcd1:2379
-| etcdUsername    | ETCD_USERNAME     | Username for RBAC (optional)               | "user"
-| etcdPassword    | ETCD_PASSWORD     | Password for the given username (optional) | "password"
-| etcdKeyFile     | ETCD_KEY_FILE     | Path to the etcd key file (optional)       | /etc/calico/key.pem
-| etcdCertFile    | ETCD_CERT_FILE    | Path to the etcd client cert (optional)    | /etc/calico/cert.pem
-| etcdCACertFile  | ETCD_CA_CERT_FILE | Path to the etcd CA file (optional)        | /etc/calico/ca.pem
+| Setting (Environment variable)     | Description                                                                            | Schema
+| ---------------------------------- | -------------------------------------------------------------------------------------- | ------
+| datastoreType (DATASTORE_TYPE)     | Indicates the datastore to use [Default: `etcdv2`] (optional)                          | kubernetes, etcdv2
+| etcdEndpoints (ETCD_ENDPOINTS)     | A comma separated list of etcd endpoints [Default: `http://127.0.0.1:2379]` (optional) | string
+| etcdUsername (ETCD_USERNAME)       | Username for RBAC, e.g. `user` (optional)                                              | string
+| etcdPassword (ETCD_PASSWORD)       | Password for the given username, e.g. `password` (optional)                            | string
+| etcdKeyFile (ETCD_KEY_FILE)        | Path to the etcd key file, e.g. `/etc/calico/key.pem` (optional)                       | string
+| etcdCertFile (ETCD_CERT_FILE)      | Path to the etcd client cert, e.g. `/etc/calico/cert.pem` (optional)                   | string
+| etcdCACertFile (ETCD_CA_CERT_FILE) | Path to the etcd CA file, e.g. `/etc/calico/ca.pem` (optional)                         | string
 
 > **NOTES**
 >
```

Comment:
Nothing to do with this change I realize, but could you fix up the typo at line 64 END->ETCD while you're updating.  Ta :-)